feed/tests/test_disaggregationtypeview.py

```diff
@@ -35,11 +36,53 @@ def test_list_disaggregationtype_normaluser(self):
 
     def test_list_disaggregationtype_normaluser_one_result(self):
         tola_user = factories.TolaUser()
-        DisaggregationType.objects.create(disaggregation_type='DisaggregationType0',
-                                          organization=tola_user.organization)
+        DisaggregationType.objects.create(
+            disaggregation_type='DisaggregationType0',
+            organization=tola_user.organization)
 
         self.request_get.user = tola_user.user
         view = DisaggregationTypeViewSet.as_view({'get': 'list'})
         response = view(self.request_get)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.data), 1)
+
+
+class DisaggregationTypeCreateViewsTest(TestCase):
+    def setUp(self):
+        self.organization = factories.Organization()
+        self.factory = APIRequestFactory()
+
+    def test_create_disaggregationtype_normaluser_missing_values(self):
+        user = factories.User(is_superuser=True, is_staff=True)
```

Comment:
`superuser` isn't `normaluser`. If it's supposed to use a superuser in the test, just change the name of the test.

Reply:
The other test is called normaluser as well but it uses a superuser